.github/workflows/ci.yml

```diff
@@ -4,18 +4,21 @@ on:
   push:
     branches:
     - '*'
-    tags:
-    - 'v*'
   pull_request:
     branches:
     - '*'
 
 jobs:
-  verify:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
-        with:
-          go-version: '1.16'
-      - run: make test
+  linter:
+    name: "Run Golang linter"
+    uses: kubewarden/kubewarden-controller/.github/workflows/golangci-lint.yml@main
```

Comment:
I see. I'm using the main because is the branch that I have the files. But you are right. This can be an issue. I'll check if it is possible to use some Github env vars or context to get the current branch and use it on this.